front-end/src/components/LeafletMap.vue

```diff
@@ -49,26 +49,43 @@ let leaflet: L.Map | undefined;
 
 const markerTable = new Map<string, Marker<null>>();
 
+const DEFAULT_LAT = 41.87213786;
+const DEFAULT_LONG = -87.62576558;
+const DEFAULT_ZOOM_OUT = 14;
+const DEFAULT_ZOOM_IN = 18;
```

Comment:
It took me a minute to understand the zoom default names. I think this should be `DEFAULT_ZOOM_LEVEL` and `HIGHLIGHT_ZOOM_LEVEL` (or something similar). As written, they feel like a default for the allowable zoom range ("you get whatever level you want, so long as it's between 14–18!"), or… something.